SatelliteQE/robottelo
# RFE Automation for syncing exported repos

SAT-15864

Automation to test syncing exported repositories

Dependent on https://github.com/SatelliteQE/fedorapeople-repos/pull/13

Steps:
1. Execute hammer command
        hammer content-import repository --organization='' --path=''
2. Assert job is succesful
3. Verify repsoitory is synced and contains rpms
